starter-code/.gitignore

```diff
@@ -0,0 +1,3 @@
+scripts/models/githubToken.js
```

Comment:
☝️  didn't work, maybe you committed `githubToken.js` before adding it to the `.gitignore`?